src/Microsoft.VisualStudio.Web.CodeGeneration.Msbuild/Target/buildCrossTargeting/Microsoft.VisualStudio.Web.CodeGeneration.Tools.targets

```diff
@@ -0,0 +1,50 @@
+﻿<!--
+ Copyright (c) .NET Foundation. All rights reserved.
+ Licensed under the Apache License, Version 2.0.
+ -->
+
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<!--
+**********************************************************************************
+Target: EvaluateProjectInfoForCodeGeneration
+
+Outputs the Project Information needed for CodeGeneration to a file.
+
+**********************************************************************************
+-->
+
+  <UsingTask TaskName="GetNearestTargetFramework" AssemblyFile="$(MicrosoftNETBuildTasksAssembly)" />
```

Comment:
O_o Awesome this works now! Curious where's the `MicrosoftNETBuildTasksAssembly` assembly?